recipes/pyads/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyads" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stlehmann/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ce00b9875a98f1d98448afb6be81978d5c37736de64ba9b9a432267a455fcbb7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pyads  # [not win]
```

Comment:
Why doesn't this get tested on Windows? There needs to be some kind of test if only to make sure the package isn't empty.

Reply:
The functionality on platforms differs here:
* Windows: pyads uses separately-installed vendor-copyrighted `.dll` library (not packaged)
* Linux/OSX: pyads compiles and uses its own open-source equivalent

End result on build server:
* Windows: import test fails; dll not found
* Linux/OSX: import succeeds

What do you think is a good forward here?

Reply:
@chrisburr as @klauer wrote a function-test on Windows is not possible because of a missing vendor-library. However I could modifiy the test to check the operating system and only test for the package on Windows while doing full checks on Linux. Would this suffice?

Reply:
This turns out to be a problem. When importing pyads on Windows without the vendor software installed I will get an error because a library is missing.